Dockerfile

```diff
@@ -6,8 +6,8 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #
-FROM mikefarah/yq as builder
-RUN apk add --no-cache bash
+FROM alpine:3.10 AS builder
```

Comment:
if you're going to rebase to a new base image, why not use ubi-minimal? https://github.com/nickboldt/containers/blob/master/ubi8-httpd-yq/Dockerfile

Reply:
Could be, I went with alpine mostly because I meant this change to be as transparent as possible. `mikefarah/yq` is based off alpine, so this is the most similar. 

Will look into moving to ubi8-minimal.

Reply:
After glancing at the additional configuration/workarounds required, I'd rather do the ubi8 switch in a separate PR.